### PR TITLE
Add stack indices to be more clear about the example. Also fix CHAINID documentation

### LINKED ISSUE #88: Add "default stack order" to docs
I always get confused whether the convention is top/leftmost stack item first or bottom/rightmost stack item first and have to remind myself by looking at an opcode whose ordering I know (`MSTORE`). It would be helpful to add a box at the top of the docs which says "leftmost item is the top of stack".

(Here's an example, `LT`, which is not super clear)
Screenshot from 2021-12-26 16-03-54

### PR DESCRIPTION
**Pull request by peroket**
Closes #88

### CODE REVIEW

File: docs/opcodes/53.mdx [hunk 1]
```diff
@@ -3,17 +3,19 @@ fork: Frontier
...
-|------:|--:|--:|
-| `0` | * | `1` |
-| `0xFFFF` | * | `0xFF` |
+| * | Input 1 | * | * | Input 2 |
+|--:|--------:|--:|--:|--------:|
+| `1` | `0` | * | `1` | `1` |
+| `2` | `0xFFFF` | * | `2` | `0xFF` |
 
 | Memory before |
 |--------------:|
```
Comment (ArthurBraud) [inline docs/opcodes/53.mdx:21]: Nit: maybe we could also add the memory slot index for consistency, but totally fine without too

Comment (peroket) [reply docs/opcodes/53.mdx:21]: There is no memory slot, only storage slot. Memory is byte indexed. Not sure how we can make that more clear, showing it byte by byte is not a nice solution I think.

File: docs/opcodes/60.mdx [hunk 2]
```diff
@@ -3,19 +3,25 @@ fork: Frontier
...
 ## Example
 
 | Code |
-|------|
+|-----:|
 | `0x60FF` |
 
-| Input | Output |
-|------:|-------:|
-| | `0xFF` |
```
Comment (ArthurBraud) [inline docs/opcodes/60.mdx:18]: nit: maybe we could add a 2nd push here to visualize that it's a LIFO (the notes are also sufficient by themselves)

File: docs/opcodes/7D.mdx [hunk 3]
```diff
@@ -3,19 +3,25 @@ fork: Frontier
...
 
 0. `value`: pushed value, aligned to the right (put in the lowest significant bytes).
 
 ## Example
 
 | Code |
-|------|
-| `0x7DFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` |
+|-----:|
+| `1` | `0x7DFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF` |
```
Comment (ArthurBraud) [inline docs/opcodes/7D.mdx:20]: No need to add the `1` here

File: docs/opcodes/53.mdx (hunk 1 above)
Comment (ArthurBraud) [reply docs/opcodes/53.mdx:21]: that's true, I agree

**Comment by dorlevi** [comment]
Perhaps we should move the top of the stack to be under Notes not above (as I guess, it is a note)
Screen Shot 2021-12-28 at 11 16 58 AM

File: docs/opcodes/7D.mdx (hunk 3 above)
Comment (peroket) [reply docs/opcodes/7D.mdx:20]: Thanks removed :-)

**Comment by peroket** [comment]
> Perhaps we should move the top of the stack to be under Notes not above (as I guess, it is a note) the screenshots show it under the existing note but obviously could be the first note if you prefer it to be on top.
> 
> Screen Shot 2021-12-28 at 11 16 58 AM

It is a note but a special kind, it's not a note about the opcode, it's a note about our representation, so I would rather keep them separate, otherwise I find it confusing.

File: docs/opcodes/60.mdx (hunk 2 above)
Comment (peroket) [reply docs/opcodes/60.mdx:18]: good idea, done

**Review by ArthurBraud — APPROVED** [review]

**Pull request merged**